packages/app-store/_utils/useAddAppMutation.ts

```diff
@@ -28,10 +29,21 @@ type UseAddAppMutationOptions = CustomUseMutationOptions & {
 
 function useAddAppMutation(_type: App["type"] | null, allOptions?: UseAddAppMutationOptions) {
   const { returnTo, ...options } = allOptions || {};
+  const router = useRouter();
+  const onErrorReturnTo = `${WEBAPP_URL}${router.asPath}`;
```

Comment:
Get the URL that the app is being installed from inside of the hook. That way we don't have to add the URL param to every place the hook is used.